### DISCUSSION TITLE
Keep podman machine running after Podman Desktop exit

### DISCUSSION THREAD

**Discussion by tony-sol** [podman-desktop/podman-desktop#9552] [Q&A] [answered]
Sorry if this was already asked (i've tried to find, but without success) - is there a config to prevent stopping podman machine on exit Podman Desktop?

**Comment by benoitf** [comment, marked as answer]
in podman settings

![image](https://github.com/user-attachments/assets/6f7319bb-ee6a-4233-baf4-e0ae0e49da98)

**Reply by tony-sol** [reply]
So, am i get this right - if i enable autostart it also enables kinda "autostop"?

**Reply by benoitf** [reply]
yes correct, the description is not clear

**Reply by tony-sol** [reply]
Not exactly sure about description, it's more like strange behaviour. Maybe it's better to add another "Autostop" option and unbound machine stopping logic from autostart?

Like, if you want to start|stop machine along with Podman Desktop - both turn on; want to completely separate machine from desktop - both turn off

In my case, i want to keep machine alive independently of desktop, so i'd turn autostart on and autostop off

If i implement this, will it be accepted?

**Reply by benoitf** [reply]
I think so, autostop being enabled by default
